python/ray/train/examples/tensorflow_linear_dataset_example.py

```diff
@@ -55,7 +55,7 @@ def train_func(config):
         # Model building/compiling need to be within `strategy.scope()`.
         multi_worker_model = build_and_compile_model(config)
 
-    dataset_pipeline = train.get_dataset_shard()
+    dataset_pipeline = train.get_dataset_shard(config.get("dataset_name", None))
```

Comment:
Hmm what's this change for?

Reply:
Oh I see this is for the test.

Reply:
Can we use the function from `v2/examples` for the test instead of the one here?

Reply:
yeah will do